json-path/src/main/java/com/jayway/jsonpath/internal/function/PathFunctionFactory.java

```diff
@@ -25,11 +25,11 @@
  */
 public class PathFunctionFactory {
 
-    public static final Map<String, Class> FUNCTIONS;
+    public static final Map<String, Class<? extends PathFunction>> FUNCTIONS;
```

Comment:
Maybe move the value/initialization to the `Configuration` class and just keep a reference here?